polygeist/tools/cgeist/Lib/utils.cc

```diff
@@ -89,10 +87,6 @@ gpu::GPUModuleOp getDeviceModule(ModuleOp Module) {
       Module.lookupSymbol(MLIRASTConsumer::DeviceModuleName));
 }
 
-llvm::raw_ostream &warning() {
-  if (SuppressWarnings)
-    return llvm::nulls();
-  return llvm::WithColor::warning();
-}
+llvm::raw_ostream &warning() { return llvm::WithColor::warning(); }
```

Comment:
Always return `warning()` now.

Reply:
Should we directly use `llvm::WithColor::warning()` and remove `mlirclang::warning()`? so is clear the intent.

Reply:
Agree. I though of doing that, but wanted to discuss beforehand. Done.